### PR TITLE
CDAP-8790 log reading fix while exception on file

### PR DESCRIPTION
**Pull request by gsps1**
JIRA : https://issues.cask.co/browse/CDAP-8790 
have tested on cluster to work.

### CODE REVIEW

**Comment by gsps1** [comment]
build running : http://builds.cask.co/browse/CDAP-RUT727-1

File: cdap-watchdog/src/main/java/co/cask/cdap/logging/write/LogLocation.java [hunk 1]
```diff
@@ -144,65 +150,61 @@ public void readLog(Filter logFilter, long fromTimeMs, long toTimeMs, int maxEve
    */
   @SuppressWarnings("WeakerAccess")
   public Collection<LogEvent> readLogPrev(Filter logFilter, long fromTimeMs, final int maxEvents) throws IOException {
-    DataFileReader<GenericRecord> dataFileReader = createReader();
-
+//    List<List<LogEvent>> logSegments = Lists.newArrayList();
```
Comment (chtyim) [inline cdap-watchdog/src/main/java/co/cask/cdap/logging/write/LogLocation.java]: Remove this comment.

**Review by chtyim — APPROVED** [review]
Just one minor comment, rest LGTM

**Comment by gsps1** [comment]
build green, merging

**Pull request merged**